docs/src/modules/spring/partials/entity-keys.adoc

```diff
@@ -0,0 +1,27 @@
+
+== Identifying the Entity
+
+In order to interact with an Entity in Kalix, we need to assign an entity type and one or more entity keys. The entity type is a unique identifier for all entities of a given type. The entity key, on the other hand, is unique per instance.
+
+In most cases, the entity key is passed as a path parameter of a REST request. The exception to the rule is when we request Kalix to auto-generate a key for us. In such a case, Kalix won't try to extract the key from the endpoint path.
+
+To define the entity type, the entity class must be annotated with `@EntityType` and have a unique and stable name assigned.
+
+The entity key can be defined in different ways.
```

Comment:
I think in this paragraph we jump between entity key and entity type several times which can make it a bit more difficult to process and refer to.. What do you think of aggregating the info into 2 bullet points? e.g:

```suggestion
In order to interact with an Entity in Kalix, we need to assign an *entity type* and one or more *entity keys*: 
- *entity type* is a unique identifier for all entities of a given type. To define the entity type, the entity class must be annotated with `@EntityType` and have a unique and stable name assigned.
- *entity key*, on the other hand, is unique per instance. In most cases, the entity key is passed as a path parameter of a REST request. The exception to the rule is when we request Kalix to auto-generate a key for us. In such a case, Kalix won't try to extract the key from the endpoint path.

The entity key can be defined in different ways, as detailed below.
```

Reply:
Bullet points are not working. Wrong syntax. I will fix it later. Let's not merge it for now.

Reply:
fixed now